var/spack/repos/builtin/packages/blaspp/package.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+# 1) The CMake options exposed by `blaspp` allow for a value called `auto`. The
+#    value is not needed here as the choice of dependency in the spec
+#    determines the appropriate flags.
+#
+# 2) BLASFinder.cmake handles most options. For `auto`, it searches all blas
+#    libraries listed in `def_lib_list`.
+#
+# 3) ?? Custom blas library can be supplied via `BLAS_LIBRARIES`.
+#
+class Blaspp(CMakePackage):
+    """BLAS++: C++ API for the Basic Linear Algebra Subroutines (University of
+    Texas)."""
+
+    homepage = "https://bitbucket.org/icl/blaspp"
+    hg       = "https://bitbucket.org/icl/blaspp"
+    maintainers = ['teonnik', 'Sely85']
+
+    version('develop', hg=hg, revision="5191c9d")
+
+    variant('gfort',
+            default=False,
+            description=('Use GNU Fortran interface. '
+                         'Default is Intel interface. (MKL)'))
+    variant('ilp64',
+            default=False,
+            description=('Use 64bit integer interface. '
+                         'Default is 32bit. (MKL & ESSL)'))
+    variant('parallel',
+            default=False,
+            description=('Use OpenMP threaded backend. '
+                         'Default is sequential. (MKL & ESSL)'))
+
+    depends_on('blas')
```

Comment:
Is mkl another (optional?) dependency of this package?

Reply:
Hmm, I am not sure about that. `mkl` is a virtual package, I am not sure whether I need `depends_on('mkl')` explicitly here or 'blas' is sufficient. Either way, the package seems to build fine.

Reply:
Just checked, `blas` is the actual dependency of this package. `intel-mkl` provides several virtual packages, including `blas` and `mkl`.